client/src/game/ui/ThreeLineButton.ts

```diff
@@ -0,0 +1,74 @@
+// Imports
+import Konva from 'konva';
+import Button from './Button';
+import FitText from './FitText';
+
+export default class ThreeLineButton extends Button {
+  textElement1: Konva.Text;
+  textElement2: Konva.Text;
+  textElement3: Konva.Text;
+
+  constructor(config: Konva.ContainerConfig) {
+    super(config);
+    this.listening(true);
+
+    // Hide the default label
+    this.textElement?.hide();
```

Comment:
should this be ! instead of ?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining
Means call hide() if not null, don't call anything otherwise

Reply:
k

Reply:
It's actually the opposite of ! :)